src/Components/ExperimentOne/ExperimentOneDisplayInformation.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import BoredAPICaller from "../../ApiTemplates/BoredAPICaller.js";
+import ExperimentOneUserForm from "./ExperimentOneUserForm.js";
+
+const ExperimentOneDisplayInformation = (props) => {
+  const [userName, setUserName] = useState("");
+  const roboHashUrl = "https://robohash.org/" + userName + ".png?size=200x200";
+
+  const [submitButtonState, setSubmitButtonState] = useState(false);
+  const [userActivityType, setUserActivityType] = useState("social");
+  const [userActivityDescription, setUserActivityDescription] = useState(
+    "Stare at wall"
+  );
+
+  const userOutputMessage =
+    "Hey " +
+    userName +
+    ". Based on what you have told me, I would suggest doing something along the lines of " +
+    userActivityType +
+    ". Therefore, maybe try " +
+    userActivityDescription.toLowerCase() +
+    ". Also look what I found below. This is how you would look in a Robot conquered world \u2728";
+
+  const [numberOfParticipants, setNumberOfParticipants] = useState(1);
+
+  const storeUserPreferencesFromApi = (
+    userActivityFromApi,
+    userActivityTypeFromApi
+  ) => {
+    setUserActivityType(userActivityTypeFromApi);
+    setUserActivityDescription(userActivityFromApi);
+  };
+
+  const storeFormData = (
+    userNameFromForm,
+    numberOfParticipants,
+    isSubmitPressed
+  ) => {
+    setUserName(userNameFromForm);
+    setSubmitButtonState(isSubmitPressed);
+    setNumberOfParticipants(numberOfParticipants);
+  };
+
+  return (
+    <div>
```

Comment:
Could use a React Fragment here... don't need a div

Reply:
Whats the advantge of using Fragment over this? Because I remember reading that you can use either but never knew the benefits of using `<>` or `<div>` or `<React.Fragment>`

Reply:
firstly, `<>` and `<React.Fragment` are identical - it's just nicer to write `<>` :) 

the main benefit of using fragments over divs is that with a div you actually end up rendering that div element in the DOM, however when you use a fragment, they're just a react thing and dont actually affect the markup that get generated. see this - top answer is from dan abramov who is a very big name in react (he created redux, create-react-app and now works on the react core team at facebook) https://stackoverflow.com/questions/47761894/why-are-fragments-in-react-16-better-than-container-divs

Reply:
That's good to know. I have indeed heard about Dan now. Sorry did not see your comment earllier